features/borrowers_view_and_sign_deed.feature

```diff
@@ -0,0 +1,280 @@
+  And borrower <3> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then the deed is digitally signed by borrower <3>
+
+@us133
+Scenario: 3 Borrowers can view deed mutltiple times, and sign once. p2
+  Given I have valid deed data with <3> borrowers
+  And I create the deed via the Deed API
+  And borrower <3> views the deed
+  And borrower <1> views the deed again
+  Then the deed is digitally signed by borrower <1>
+  And borrower <2> views the deed again
+  And borrower <3> views the deed again
+  Then the deed is digitally signed by borrower <3>
+  And borrower <2> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then I verify borrower <1> has signed the deed
+  Then the deed is digitally signed by borrower <2>
+  And borrower <1> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then I verify borrower <3> has signed the deed
+
+@us133
+Scenario: 4 Borrowers can view a deed
+  Given I have valid deed data with <4> borrowers
+  And I create the deed via the Deed API
+  And borrower <2> views the deed
+  And borrower <4> views the deed again
+  And borrower <1> views the deed again
+  Then borrower <3> views the deed again
+
+@us133
+Scenario: 4 Borrowers can view and sign a deed. p1
+  Given I have valid deed data with <4> borrowers
+  And I create the deed via the Deed API
+  And borrower <3> views the deed
+  Then the deed is digitally signed by borrower <3>
+  And borrower <2> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then the deed is digitally signed by borrower <2>
+  And borrower <1> views the deed again
+  Then I verify borrower <2> has signed the deed
+  Then I verify borrower <3> has signed the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <4> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <2> has signed the deed
+  Then I verify borrower <3> has signed the deed
+  Then the deed is digitally signed by borrower <4>
+
+@us133
+Scenario: 4 Borrowers can view and sign a deed. p2
+  Given I have valid deed data with <4> borrowers
+  And I create the deed via the Deed API
+  And borrower <1> views the deed
+  Then the deed is digitally signed by borrower <1>
+  And borrower <3> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then the deed is digitally signed by borrower <3>
+  And borrower <4> views the deed again
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <3> has signed the deed
+  Then the deed is digitally signed by borrower <4>
+  And borrower <2> views the deed again
+  Then I verify borrower <3> has signed the deed
+  Then I verify borrower <1> has signed the deed
+  Then I verify borrower <4> has signed the deed
+  Then the deed is digitally signed by borrower <2>
```

Comment:
A lot of these scenarios end with the final borrower signing the deed but there is no extra check that the deed is then viewed and all signatures have been applied to the deed.